wordcount.c

```diff
@@ -0,0 +1,25 @@
+//Albert Zhang
```

Comment:
Nice job! the only thing is once it sees any "#" it counts that as the end, when it could be part of the text, so it cuts of the program short.